docker/opensuse-42.2/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN zypper --no-color --non-interactive ref -f && \
     zypper --no-color --non-interactive install --no-recommends python gcc \
             gcc-c++ git chrpath make wget python-xml diffstat makeinfo \
             python-curses patch socat libSDL-devel tar which unzip \
-            python3 python3-curses glibc-locale syslinux
+            python3 python3-curses glibc-locale syslinux python3-devel \
+            python3-six python3-netifaces python3-unittest-xml-reporting
```

Comment:
The commit message confuses me. What are the tests needing these and why these are not needed already now?